frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -24,6 +36,21 @@ describe("collection permissions", () => {
                 cy.signIn(user);
               });
 
+              describe("create dashboard", () => {
+                it("should offer to save dashboard to a currently opened collection", () => {
+                  cy.visit("/collection/root");
+                  cy.get("[class*=CollectionSidebar]").within(() => {
+                    cy.findByText("First collection").click();
+                    cy.findByText("Second collection").click();
+                  });
+                  cy.icon("add").click();
+                  cy.findByText("New dashboard").click();
+                  cy.get(".Modal")
+                    .findByText("Second collection")
+                    .should("exist");
```

Comment:
```suggestion
                  cy.get(".AdminSelect").findByText("Second collection");
```
No need to use explicit `.should("exist")` assertion. `cy.findByText()` fails if it doesn't find a string and/or if it finds multiple strings so implicit assertion is always preferred.

Btw, I suggested we check for `.AdminSelect` because it is more specific and it makes it more clear to anyone who reads this test later what was the intention behind this assertion.

Reply:
Thanks for your feedback! Applied the change in my own commit not to trigger the CI once again